images/android-ndk-r17c-buck

```diff
@@ -0,0 +1,37 @@
+FROM mbgl/android-base AS buck
+
+RUN set -eu \
+ && apt-get update \
+ && apt-get -y install ant \
+ && rm -rf /var/lib/apt/lists/*
+
+RUN set -eu \
+ && git clone --depth=1 https://github.com/facebook/buck.git
```

Comment:
Is there a need to pick particular version / tag?

Reply:
Good call; I looked at the releases and counter to what https://buckbuild.com/setup/getting_started.html says, the Github release page actually has .deb packages of the releases.